requirements/key-revocation.md

```diff
@@ -1,6 +1,6 @@
 # Key Revocation
 
-One of the goals of Notary v2 is to build in solutions for key revocation that are easy to use and ensure that users will always use non-compromised keys.
+One of the goals of Notary is to build in solutions for key revocation that are easy to use and ensure that users will always use non-compromised keys.
```

Comment:
The term `Notary` is ambiguous. Currently, if nothing is referred, `Notary` is well known as `Docker Notary` or simply `Docker Content Trust`.